common/src/main/cpp/include/extensions/openxr_htc_passthrough_extension_wrapper.h

```diff
@@ -0,0 +1,123 @@
+/**************************************************************************/
+/*  openxr_htc_passthrough_extension_wrapper.h                            */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
```

Comment:
This section needs to be updated to reflect the style used in this repo.
Or are you leaving it like this because it originated from the Godot repo?

Reply:
I copied it from the FB one, so I guess we haven't changed it in all classes?

Reply:
We forgot to do so when migrating the class over. Can you update the fb one as well.